WORKSPACE.bzlmod

```diff
@@ -0,0 +1,23 @@
+# When Bzlmod is enabled, this file replaces the content of the original WORKSPACE
+# and makes sure no WORKSPACE prefix or suffix are added when Bzlmod is enabled.
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+_disco_to_proto3_converter_version = "74c0ede1e6871c878de6c3cab3ef152280c6518f"
```

Comment:
Are those only dev dependencies?

Reply:
they aren't, but i didn't handle all the different language specific dependencies here, and this one is one that was easy enough to throw here so that a few more targets built when testing the integration in this repo.